pkg/scsi/scsi_pr.go

```diff
@@ -255,11 +256,19 @@ func (op *SCSISimpleReservationOperator) DeleteAndRemoveReservation(tgtName stri
 
 func (op *SCSISimpleReservationOperator) RemoveReservation(tgtName string, devUUID uint64, res *api.SCSIReservation) {
 	var (
-		i      int = -1
-		tmpRes *api.SCSIReservation
+		LURes     *SCSILUReservation
+		targetRes SCSILUReservationMap
+		tmpRes    *api.SCSIReservation
+		i         int = -1
+		ok        bool
 	)
-
-	resArray := op.GetReservationList(tgtName, devUUID)
+	if targetRes, ok = op.targetReservations[tgtName]; !ok {
+		return
+	}
+	if LURes, ok = targetRes[devUUID]; !ok {
+		return
+	}
+	resArray := LURes.Reservations
```

Comment:
Why using this code block? It looks like that `GetReservationList` function do the same work with yours.